### PR TITLE
added a new hardcoded servlet

### PR DESCRIPTION
**Pull request by priyalsuneja**
This is step 2 of the AboutMe Guided Project

### CODE REVIEW

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -42,9 +42,11 @@ function showMessageFormIfViewingSelf() {
             loginStatus.username == parameterUsername) {
           const messageForm = document.getElementById('message-form');
           messageForm.classList.remove('hidden');
+          document.getElementById('about-me-form').classList.remove('hidden');
+          }
         }
       });
-}
```
Comment (ginellegaisano) [inline src/main/webapp/js/user-page-loader.js:47]: I think this is still needed to match the brace on line 35.

File: src/main/webapp/js/user-page-loader.js [hunk 2]
```diff
@@ -95,4 +97,19 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
-}
\ No newline at end of file
+}
+
+function fetchAboutMe(){
```
Comment (ginellegaisano) [inline src/main/webapp/js/user-page-loader.js:102]: nit: needs whitespace between () and {.

File: src/main/webapp/js/user-page-loader.js (hunk 1 above)
Comment (priyalsuneja) [reply src/main/webapp/js/user-page-loader.js:47]: oops, thank you!

File: src/main/webapp/js/user-page-loader.js (hunk 2 above)
Comment (priyalsuneja) [reply src/main/webapp/js/user-page-loader.js:102]: oh, did not know that. Thank you!

**Comment by priyalsuneja** [comment]
I corrected all the things that @ginellegaisano mentioned and added Step 5

**Comment by priyalsuneja** [comment]
Officially finished the AboutMe guided project :)

File: src/main/webapp/js/user-page-loader.js [hunk 3]
```diff
@@ -65,6 +65,7 @@ function fetchMessages() {
         messages.forEach((message) => {
           const messageDiv = buildMessageDiv(message);
           messagesContainer.appendChild(messageDiv);
+
```
Comment (ginellegaisano) [inline src/main/webapp/js/user-page-loader.js]: nit: remove whitespace

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 4]
```diff
@@ -79,4 +81,35 @@ public void storeMessage(Message message) {
...
+    if (userEntity == null) {
+      return null;
+    }
+
+    String aboutMe = (String) userEntity.getProperty("aboutMe");
+    User user = new User(email, aboutMe);
+
+    return user;
+  }
+
```
Comment (ginellegaisano) [inline src/main/java/com/google/codeu/data/Datastore.java]: nit: remove newline between last method and class

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 5]
```diff
@@ -61,8 +63,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     String userEmail = userService.getCurrentUser().getEmail();
-    String aboutMe = request.getParameter("about-me");
-
+    String aboutMe = Jsoup.clean(request.getParameter("about-me"), Whitelist.none());
+
```
Comment (ginellegaisano) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: nit: remove whitespace

**Comment by ginellegaisano** [comment]
Hey! Just a few style nits from me. Otherwise, looks good.

**Review by ginellegaisano — APPROVED** [review]

**Pull request merged**